USAGE.md

```diff
@@ -0,0 +1,240 @@
+# FHEM Webuntis Module - Usage Guide
+
+## Overview
+
+The `69_Webuntis.pm` module retrieves timetable data from the Webuntis school cloud service for use in the FHEM home automation system.
+
+## Requirements
+
+### Required Perl Modules
+
+The following Perl modules must be installed:
+
+```bash
+# On Debian/Ubuntu:
+sudo apt-get install -y libdatetime-perl libdatetime-format-strptime-perl libdigest-sha-perl
+
+# On other systems, use CPAN:
+cpan DateTime DateTime::Format::Strptime Digest::SHA
+```
+
+Additionally, one of these JSON modules is required:
+- `JSON::XS` (recommended for performance)
+- `JSON::PP` (pure Perl fallback)
+- `Cpanel::JSON::XS`
+- `JSON::MaybeXS`
+
+### FHEM-Specific Dependencies
+
+The module requires these FHEM internal modules (automatically available in FHEM):
+- `HttpUtils`
+- `FHEM::Meta`
+- `GPUtils`
+- `DevIo`
+- `FHEM::Core::Authentication::Passwords`
+
+## Installation
+
+1. Copy `FHEM/69_Webuntis.pm` to your FHEM modules directory (typically `/opt/fhem/FHEM/`)
```

Comment:
We have also a way to inlcude the repository in fhem and have the module being part of automatic updates etc.
please change or add this possibility with instructions.
example fhem command: 
update add https://raw.githubusercontent.com/tobi01001/......../main/controls_webuntis.txt